blog/2024-10-30-2024-kwdc.md

```
@@ -41,85 +41,106 @@ Swift에서는 View protocol의 body 생성자를 통해 화면에 표시할 구
 
 ### 색상을 통한 표현
 
-색상은 감정을 불러 일으키는 효과가 있음, 하지만 이러한 색상에 대한 이미지도 나라별로 큰 차이가 있음, 이를 고려한 디자인 필요
+색상은 감정을 불러 일으키는 효과가 있다. 하지만 이러한 색상에 대한 이미지도 나라별로 큰 차이가 있으므로, 이를 고려한 디자인 필요하다. 아래 예시를 통해 보자.
 
-ex) 녹색과 빨강에 대한 인식
+#### 녹색과 빨강에 대한 인식
 
-녹색 - 동양권: 친환경, 서양권: 행운
-빨강 - 동양권: 행운, 텍스트에 사용 시 불길한 이미지, 남미: 사랑, 서양권: 폭력
+* 녹색 - 동양권: 친환경, 서양권: 행운
```

Markdownlint

Expected: dash; Actual: asterisk
Unordered list style

(MD004, ul-style)